doc/make.py

```
@@ -198,11 +206,6 @@ def _add_redirects(self):
                     # sphinx specific stuff
                     title = "this page"
 
-                if os.path.exists(path):
-                    raise RuntimeError(
-                        f"Redirection would overwrite an existing file: {path}"
-                    )
```

Thanks, I regularly ignored this error .. ;)